applications/GeoMechanicsApplication/tests/cpp_tests/test_prescribed_time_incrementor.cpp

```diff
@@ -0,0 +1,143 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//                   Anne van de Graaf
+//
+
+#include "testing/testing.h"
+#include "custom_workflows/prescribed_time_incrementor.h"
+#include "custom_workflows/time_step_end_state.hpp"
+
+using namespace Kratos;
+
+namespace
+{
+
+TimeStepEndState MakeConvergedEndState()
+{
+    auto result = TimeStepEndState{};
+    result.convergence_state = TimeStepEndState::ConvergenceState::converged;
+    return result;
+}
+
+}
+
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(NoNextTimeStepWhenPrescribedTimeIncrementorHasEmptyList, KratosGeoMechanicsFastSuite)
+{
+    std::vector<double> increments;
```

Comment:
In all tests, the increments vector can be const std::vector<double> :)

Reply:
Good catch. In fact, there were a few more variables that could also be made `const`. I have changed those as well.